--- dumpstate_30.rc	2025-03-15 13:47:23.976294800 +0000
+++ dumpstate_30_edited.rc	2025-03-15 13:47:23.976294800 +0000
@@ -45,4 +45,10 @@
 service bugreportm /system/bin/dumpstate -m -d -z
     class main
     disabled
-    oneshot
\ No newline at end of file
+    oneshot
+
+# horizon changes:
+on property:sys.boot_completed=1
+    stop dumpstate
+    stop dumpstatez
+    stop bugreportd
+    stop bugreportm
\ No newline at end of file
